src/VideoManager/VideoManager.h

```diff
@@ -28,6 +28,13 @@ class VideoSettings;
 class Vehicle;
 class Joystick;
 
+enum {
```

Comment:
Add LM prefix to mark that it is related to our project:
```c++
struct LMTargetAction {
    LM_TARGET_UNUSED=0,
    LM_TARGET_LOCK,
    LM_TARGET_FOLLOW,
    LM_TARGET_CANCEL,
};
```

Reply:
done